### ISSUE TITLE
[bug] pushing a route w/o navigation bar causes red screen

### ISSUE THREAD

**Issue by grabbou** [exponentjs/ex-navigation#16]
When route A has navigationBar and route B doesn't have it (no static route set), I am getting the following exception:

<img width="350" alt="screen shot 2016-08-03 at 13 19 03" src="https://cloud.githubusercontent.com/assets/2464966/17363799/e7345b20-597c-11e6-9def-a235de08de52.png">

For now, I did what I usually do - one container Navigator and two nested ones, one w/o header for onboarding scenes and 2nd one, called home, with navigation bar.

**Comment by chirag04** [comment]
setting navigationBar visible false on route B should work.

```
Component.route = {
  navigationBar: {
    visible: false,
  },
};
```

**Comment by grabbou** [comment]
Confirmed. Works. Not sure if you want to fix the bug though (or shall we fix it).

**Comment by chirag04** [comment]
i think it's reasonable to assume all screens to have a navbar. users can opt in to not render a navbar using `visible: false`. 

I would not consider it a bug really :) cc @brentvatne

**Comment by brentvatne** [comment]
Every route needs to have `static route` defined on it right now. We can change that, assigned that to me

**Comment by grabbou** [comment]
Ah, sorry, didn't know that it was required. Makes sense now.

**Comment by chirag04** [comment]
@brentvatne I think we should keep this as is `Every route needs to have static route defined on it right now.`.  it could get confusing and hard to reason out in future when you see a screen component without a `route` defined.

I know lack of documentation can make it hard to discover but once we have that this shouldn't a problem.

**Comment by brentvatne** [comment]
@chirag04 - fair, we could at least warn in this case though, rather then showing a hard error

**Comment by skevy** [comment]
I actually fixed this specific bug in v1.4. Going to close this...please re-open if there's an issue.
